app/backend/case/views/case_count_view.py

```
@@ -149,6 +149,11 @@ def get_data(self, request):
                 ),
                 "case_count": item["case_count"],
                 "death_count": item["death_count"],
+                # todo: must not use hardcoded outbreak threshold
```

Pylint (3.3.7)

[warning] todo: must not use hardcoded outbreak threshold

(W0511)